test cases/frameworks/38 gettext extractor/src/lib3/meson.build

```
@@ -0,0 +1,7 @@
+touch = find_program('touch', required: false, disabler: true)
+
+foo = custom_target('foo',
+  output: 'foo.c',
+  command: [touch, '@OUTPUT@'],
+  )
+i18n.xgettext('test', foo)
```

```suggestion
i18n.xgettext('test', foo[0])
```

So we can test the indexing too.